manifests/candlepin.pp

```diff
@@ -0,0 +1,54 @@
+# Katello configuration for pulp
+class katello::candlepin(
+  Variant[Array[String], String] $user_groups = $::katello::user_groups,
+  String $oauth_key = $::katello::oauth_key,
+  String $oauth_secret = $::katello::oauth_secret,
+  String $deployment_url = $::katello::deployment_url,
+  String $db_host = $::katello::candlepin_db_host,
+  Optional[Integer[0, 65535]] $db_port = $::katello::candlepin_db_port,
+  String $db_name = $::katello::candlepin_db_name,
+  String $db_user = $::katello::candlepin_db_user,
+  String $db_password = $::katello::candlepin_db_password,
+  Boolean $db_ssl = $::katello::candlepin_db_ssl,
+  Boolean $db_ssl_verify = $::katello::candlepin_db_ssl_verify,
+  Boolean $manage_db = $::katello::candlepin_manage_db,
+  String $qpid_hostname = $::katello::qpid_hostname,
+) {
+  include ::certs::qpid
```

Comment:
Actually we use the variables from this class (```$qpid_ssl_{cert,key}```) and subscribe to it so I think it's better to include it here as well. In fact, I'm going to add ```include ::certs``` because we use those vars as well.